questdb-rs/src/ingress/mod.rs

```diff
@@ -2092,6 +2335,24 @@ impl SenderBuilder {
         Ok(self)
     }
 
+    #[cfg(feature = "ilp-over-http")]
+    /// Disables automatic line protocol version validation for ILP-over-HTTP.
+    ///
+    /// - When set to `"off"`: Skips the initial server version handshake and disables protocol validation.
+    /// - When set to `"on"`: Keeps default validation behavior (recommended).
+    ///
+    /// Please ensure client's default version ([`LINE_PROTOCOL_VERSION_V2`]) or
+    /// explicitly set protocol version exactly matches server expectation.
+    pub fn disable_line_protocol_validation(mut self) -> Result<Self> {
```

Comment:
Replace with `pub fn protocol_version(mut self, vers: ProtocolVersion)`, overriding the default when called.